pkg/diagnostics/http_monitoring.go

```diff
@@ -113,19 +116,30 @@ func (h *httpMetrics) IsEnabled() bool {
 	return h != nil && h.enabled
 }
 
-func (h *httpMetrics) ServerRequestCompleted(ctx context.Context, method string, status string, reqContentSize, resContentSize int64, elapsed float64) {
+func (h *httpMetrics) ServerRequestCompleted(ctx context.Context, method, path, status string, reqContentSize, resContentSize int64, elapsed float64) {
```

Comment:
Consider creating a struct for the passed in parameters as we have a few now.

Reply:
This is the same number of arguments that this method had in 1.11, and i don't see them changing (as that would mean adding even more cardinality). If needed, can be done separately.